containers-toolkit/Public/AllToolsUtilities.psm1

```diff
@@ -38,17 +38,17 @@ function Install-ContainerTools {
         [string]
         [ValidateNotNullOrEmpty()]
         [parameter(HelpMessage = "ContainerD version to use")]
-        $ContainerDVersion = (Get-ContainerdLatestVersion),
+        $ContainerDVersion = "latest",
```

Comment:
"latest" is repeated in line 41, 46 and 51 there isnt away create variable for it

Reply:
PS evaluates default parameter values when the function is defined. If we set a variable e.g `DEFAULT_VERSION = latest`, this will not be available in the scope during the function definition. The other option is to have a method `Get-DefaultVersion {return "latest"}`. This may add more complexity when debugging.